Makefile

```diff
@@ -36,3 +37,15 @@ release: iso get_gh-release
 	cp $(BUILD_DIR)/minishift-b2d.iso release/
 	$(BUILD_DIR)/gh-release checksums sha256
 	$(BUILD_DIR)/gh-release create minishift/minishift-b2d-iso $(VERSION) master v$(VERSION)
+
+$(BIN_DIR)/minishift:
+	@echo "Downloading minishift binary..."
+	@mkdir -p $(BIN_DIR)
+	@cd $(BIN_DIR) && \
+	curl -LO --progress-bar https://github.com/minishift/minishift/releases/download/v1.0.0-beta.2/minishift-1.0.0-beta.2-linux-amd64.tgz && \
```

Comment:
@budhrg hardcoded version here doesn't look good for minishift can we have some kind of script which will always retrieve latest available tarball. since we are using avocado so we assuming python would be there and so request module.

Check out https://gist.github.com/praveenkumar/f5e26a20f2b6a6640b86bc2934371bf9 if that make sense.